### PR TITLE
Adding timeout option for create server

### PR DESCRIPTION
**Pull request by cesarRockMachine**

### CODE REVIEW

File: lib/chef/knife/softlayer_server_create.rb [hunk 1]
```diff
@@ -248,6 +248,12 @@ class SoftlayerServerCreate < Knife
              :short => "-u USERDATA",
              :long => "--user-data USERDATA",
              :description => "Optional user data to pass on to SoftLayer compute instance"
+      
+      option :wait_for_timeout,
+             :short => "-t VALUE",
+	     :long => "--wait-for-timeout VALUE",
```
Comment (allmightyspiff) [inline lib/chef/knife/softlayer_server_create.rb]: add a space to make these line up.

File: lib/chef/knife/softlayer_server_create.rb [hunk 2]
```diff
@@ -310,7 +316,7 @@ def run
         if config[:private_network_only] || config[:use_private_network]
           instance.ssh_ip_address = Proc.new {|server| server.private_ip_address }
         end
-        progress Proc.new { instance.wait_for { ready? and sshable? } }
+        progress Proc.new { instance.wait_for(timeout=config[:wait_for_timeout]) { ready? and sshable? } }
```
Comment (allmightyspiff) [inline lib/chef/knife/softlayer_server_create.rb]: if wait_for_timeout isn't specified, will wait_for still work properly?

Comment (cesarRockMachine) [reply lib/chef/knife/softlayer_server_create.rb]: @allmightyspiff Thanks! Christopher. I will check your points

**Comment by cesarRockMachine** [comment]
Hi @allmightyspiff ! Could you checkout this? Thanks a lot!

**Pull request merged**